docs/docs/guides/labs/components/components-etl-pipeline-tutorial.md

```diff
@@ -49,14 +49,30 @@ See the [`duckdb`](https://duckdb.org/docs/installation/?version=stable&environm
 
 ### 2. Scaffold a new project
 
-After installing dependencies, scaffold a components-ready project:
+After installing dependencies, scaffold a components-ready project. The flow for scaffolding a project will depend on your package manager/environment management strategy.
 
-<CliInvocationExample path="docs_snippets/docs_snippets/guides/components/index/2-scaffold.txt" />
-
-The `dg scaffold project` command builds a project at `jaffle-platform` and initializes a new Python
-virtual environment inside it. When you use `dg`'s default environment management behavior, you won't need to worry about activating this virtual environment yourself.
+<Tabs>
+    <TabItem value="uv" label="`uv` (global `dg`)">
+        Ensure you have `dg` installed globally as a `uv` tool. Then run: 
+        <CliInvocationExample path="docs_snippets/docs_snippets/guides/components/index/2-a-uv-scaffold.txt" />
+        The `dg init` command builds a project at `jaffle-platform`. We passed the `--uv-sync` option, which tells `dg` to immediately run `uv sync` after scaffolding to create a virtual environment and install the dependencies listed in `pyproject.toml`. Now lets enter the directory and activate the virtual environment:
```

Comment:
I do not think the tutorial should pass `--uv-sync`. The default should be the right first time behavior, and we want to lightly "teach" the user how this stuff works.